Documentation/security/policy/language.rst

```diff
@@ -305,12 +299,6 @@ have ``external:yes`` set as the label.
 
         .. literalinclude:: ../../../examples/policies/l3/service/service-labels.json
 
-Limitations
-~~~~~~~~~~~
-
-``toServices`` statements must not be combined with ``toPorts`` statements in the
-same rule. If a rule combines both these statements, the policy is rejected.
-
```

Comment:
Are there test cases to demonstrate that such rules will now work? I understand that the sanitization step has been removed but I couldn't tell whether we have actually tested this case to ensure it works.

Reply:
> Are there test cases to demonstrate that such rules will now work?

Currently, this test case has not been integrated into the e2e. However, I can manually test it locally and include the test results in the description of this PR.

Reply:
That would be awesome thanks! :pray:

Reply:
@joestringer I've included the test results in the PR description. I believe the test results are as expected.